### ISSUE TITLE
Post Settings: Password field is black is black font on black background in Dark Mode

### ISSUE THREAD

**Issue by startuptester** [wordpress-mobile/WordPress-iOS#15768]
### Expected behavior
In Dark Mode, the password option in Post Settings is black font on a black background

### Actual behavior
![image](https://user-images.githubusercontent.com/66104034/106670776-b73b7d00-656a-11eb-88f6-a7ca4caaab9e.png)

### Steps to reproduce the behavior

1. Open Post Settings in Dark Mode
2. Choose Password Protected in the Visibility options
